### DISCUSSION TITLE
TimescaleDB

### DISCUSSION THREAD

**Discussion by Rajakavitha1** [dbeaver/dbeaver#17098] [Q&A]
Hi, 
I would like to contribute to DBeaver wiki by adding information about connecting to TimescaleDB and using TimescaleDB.
Please let me know where I can find the contribution guidelines and how I can create a new wiki page for TimescaleDB.

Regards,
Rajie

**Comment by LonwoLonwo** [comment]
Hello @Rajakavitha1 

You can provide info here in this discussion.

For now, we do not have particular databases section in our wiki, but maybe someday, we will add it. 
Then your info will be beneficial.

**Reply by Rajakavitha1** [reply]
Thank you @LonwoLonwo !!!

**Reply by Rajakavitha1** [reply]
Hi @LonwoLonwo !!! [TimescaleDB documentation](https://docs.timescale.com/timescaledb/latest/how-to-guides/connecting/dbeaver/) has detailed information about using DBeaver to connect to TimescaleDB. I was wondering if we could add a link to this documentation in the DBeaver wiki. Might help users who may be looking for particular database information in DBeaver wiki.  Maybe a "More Information" or "Related Information" section with a link to the documentation in https://github.com/dbeaver/dbeaver/wiki/Create-Connection?
